### PR TITLE
Top balances / rich list API endpoint

### PR DESCRIPTION
**Pull request by zone117x**
Closes https://github.com/blockstack/blockstack-explorer/issues/189

Implements a query and API endpoint to retrieve accounts with the highest Stacks balances. 

API endpoint is `/api/v2/top-balances`. It returns 250 entries by default. The count can be specified with a query param, e.g. `/api/v2/top-balances?count=10`. The API controller sets a max count limit of 500.

Sample of the returned data:
```json
[
  {
    "stxAddress": "SP21HJA5D2N1ZDM9ZGTNQ5WYZFG6TTWN9GX2GTFWQ",
    "btcAddress": "1CxBfCcQXUfMbnavHmNp1fnhov4Jx3RW4Y",
    "balance": "39253561.000000",
    "distribution": 9.199323
  },
  {
    "stxAddress": "SP13HN8N68CFASWYBX6C78XP5HF50EHJENRYQHJ7S",
    "btcAddress": "17UxtH36S3AFM3odE8MQepqMFhNR1sKUWb",
    "balance": "18470754.944446",
    "distribution": 4.32874
  },
  {
    "stxAddress": "SP3WV3VC6GM1WF215SDHP0MESQ3BNXHB1N6TPB70S",
    "btcAddress": "1PmGQQgsE9dFHDVxPudQZ4YoqTonswgY1q",
    "balance": "16736152.796477",
    "distribution": 3.922225
  }
]
```

`balance` is the account's unlocked balance, and `distribution` is a percentage of the total unlocked supply. 

The query uses a `top-balances` aggregator, with a cache expiry of 10 minutes.

### CODE REVIEW

File: lib/aggregators/top-balances.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  static expiry() {
+    return 10 * 60; // 10 minutes
+  }
+
+  static async setter(count: number): Promise<TopBalanceAccount[]> {
+    const { unlockedSupply } = await getUnlockedSupply();
+    const topBalances = await getTopBalances(count);
+    return topBalances.map(account => ({
```
Comment (hstove) [inline lib/aggregators/top-balances.ts]: Can we also get a `stacksAddress` attribute in here? `c32check.b58ToC32(account.address)`

File: test/aggregators/stacks-address.test.ts [hunk 2]
```diff
@@ -1,9 +1,20 @@
 import '../setup';
 import BN from 'bn.js';
 import StacksAddress from '../../lib/aggregators/stacks-address';
+import TopBalancesAggregator, { TopBalanceAccount } from '../../lib/aggregators/top-balances';
 import TotalSupplyAggregator, { TotalSupplyResult } from '../../lib/aggregators/total-supply';
 
 
+test('get top balances', async () => {
```
Comment (hstove) [inline test/aggregators/stacks-address.test.ts]: Same thing here as the other PR - let's make a separate test file for each aggregator that we test. It's just cleaner and makes it easier to only run tests for a single module.

File: lib/aggregators/top-balances.ts (hunk 1 above)
Comment (zone117x) [reply lib/aggregators/top-balances.ts]: Done

File: test/aggregators/stacks-address.test.ts (hunk 2 above)
Comment (zone117x) [reply test/aggregators/stacks-address.test.ts]: Done

**Pull request merged**